sqlmesh/core/engine_adapter/base.py

```diff
@@ -790,14 +792,18 @@ def fetchall(
         self.execute(query, ignore_unsupported_errors=ignore_unsupported_errors)
         return self.cursor.fetchall()
 
-    def _fetch_native_df(self, query: t.Union[exp.Expression, str]) -> DF:
+    def _fetch_native_df(
+        self, query: t.Union[exp.Expression, str], normalize_identifiers: bool = True
+    ) -> DF:
         """Fetches a DataFrame that can be either Pandas or PySpark from the cursor"""
-        self.execute(query)
+        self.execute(query, normalize_identifiers=normalize_identifiers)
         return self.cursor.fetchdf()
 
-    def fetchdf(self, query: t.Union[exp.Expression, str]) -> pd.DataFrame:
+    def fetchdf(
+        self, query: t.Union[exp.Expression, str], normalize_identifiers: bool = True
```

Comment:
i think this should by default be false

Reply:
Ditto

Reply:
Do we need to keep this flag at all?

Reply:
well, how do do you set it to true then if the class uses it internally?

Reply:
yea that's a good point, I assumed we'd just use `execute` directly but that's not necessarily the case

Reply:
converted to false